### PR TITLE
Added functional edit and delete buttons to displayApplicants page.

### PR DESCRIPTION
**Pull request by alexjennings56**

### CODE REVIEW

File: public/js/deleteApplicant.js [hunk 1]
```diff
@@ -2,21 +2,24 @@ window.addEventListener('load', (event) => {
...
-        return response.json()
-    }).then((responseData) => {
-        document.cookie = `response=${responseData.message}`;
-        window.location.reload(true)
-    })
+let deleteButtons = document.querySelectorAll('.deleteBtn');
+deleteButtons.forEach(deleteBtn => {
+    deleteBtn.addEventListener('click', (e) => {
+        let id = e.target.dataset.id;
+        console.log(id);
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:9]: Remove this please.

File: public/js/deleteApplicant.js [hunk 2]
```diff
@@ -2,21 +2,24 @@ window.addEventListener('load', (event) => {
...
-    }).then((response) => {
-        return response.json()
-    }).then((responseData) => {
-        document.cookie = `response=${responseData.message}`;
-        window.location.reload(true)
-    })
+let deleteButtons = document.querySelectorAll('.deleteBtn');
+deleteButtons.forEach(deleteBtn => {
+    deleteBtn.addEventListener('click', (e) => {
+        let id = e.target.dataset.id;
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:8]: Constant?

File: public/js/deleteApplicant.js [hunk 3]
```diff
@@ -2,21 +2,24 @@ window.addEventListener('load', (event) => {
...
+            method: 'DELETE',
+            body: JSON.stringify({"id": id}),
+            headers: {
+                "Content-Type": "application/json"
+            }
+        }).then((response) => {
+            return response.json()
+        }).then((responseData) => {
+            document.cookie = `response = ${responseData.message}`;
+            window.location.reload(true)
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:20]: What does `true` mean in this case? Can't see anything regarding it on the documentation. Also, missing semi-colon.

File: public/js/deleteApplicant.js [hunk 4]
```diff
@@ -2,21 +2,24 @@ window.addEventListener('load', (event) => {
...
+        })
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:21]: Missing semi-colon.

File: public/js/deleteApplicant.js [hunk 5]
```diff
@@ -2,21 +2,24 @@ window.addEventListener('load', (event) => {
...
+        let id = e.target.dataset.id;
+        console.log(id);
+        fetch('/api/deleteApplicant', {
+            method: 'DELETE',
+            body: JSON.stringify({"id": id}),
+            headers: {
+                "Content-Type": "application/json"
+            }
+        }).then((response) => {
+            return response.json()
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:17]: Missing semi-colon.

File: src/Controllers/EditApplicantController.php [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+     * @param $applicantModel
+     */
+    public function __construct($applicantModel)
+    {
+        $this->applicantModel = $applicantModel;
+    }
+
+    public function __invoke(Request $request, Response $response, array $args)
+    {
+        var_dump('success'); // this is a temporary thing to check that it works
```
Comment (bradleypurdue) [inline src/Controllers/EditApplicantController.php:25]: Is this still meant to be here?

File: public/js/deleteApplicant.js (hunk 3 above)
Comment (alexjennings56) [reply public/js/deleteApplicant.js:20]: we didn't write that so no idea.

File: public/js/deleteApplicant.js [hunk 7]
```diff
@@ -1,35 +1,37 @@
-window.addEventListener('load', (event) => {
+window.addEventListener('load', (event)     => {
```
Comment (bradleypurdue) [inline public/js/deleteApplicant.js:1]: Random tab character.

**Pull request merged**